i18n/ru-RU.js

```diff
@@ -27,12 +27,12 @@ module.exports = {
         slug: 'examples'
       },
       {
-        name: 'Resources',
+        name: 'Ресурсы',
         icon: 'resources',
         slug: 'resources'
       },
       {
-        name: 'Блог',
+        name: 'Новости',
```

Comment:
```suggestion
        name: 'Блог',
```
The 'blog' section is not only news, so I'd suggest keeping the original translation.